### PR TITLE
Add soemJavaNativeLibrary and support cross compilation for aarch64

### PR DESCRIPTION
**Pull request by ds58**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -17,8 +17,9 @@ ihmc {
 app.entrypoint("SlaveInfo", "us.ihmc.etherCAT.master.SlaveInfo")
 
 mainDependencies {
-   api("us.ihmc:soem:1.4.0-ihmc1")
-   api("us.ihmc:soem-platform-linux:1.4.0-ihmc1")
+   api("us.ihmc:soem:1.5.0")
+   api("us.ihmc:soem-platform-linux-x86_64:1.5.0")
+   api("us.ihmc:soem-platform-linux-arm64:1.5.0")
```
Comment (ds58) [inline build.gradle.kts:22]: Will have to publish these

File: readme.md [hunk 2]
```diff
@@ -131,14 +131,17 @@ The gradle build files for the SOEM wrapper and native libraries are created by
...
-- make
-- ../gradlew publishToMavenLocal -Ptarget=JAVA
-- ../gradlew publishToMavenLocal -Ptarget=PLATFORM
+```
+cd ihmc-ethercat-master
+mkdir build
+cd build
+cmake -DCMAKE_BUILD_TYPE=Release ..
+# Cross compile for arm64 using the toolchain
+# cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_TOOLCHAIN_FILE=../linux-aarch64-toolchain.cmake ..
```
Comment (reesep5) [inline readme.md:140]: I assume this means we need to run this on any arm64 machine?

File: build.gradle.kts (hunk 1 above)
Comment (calvertdw) [reply build.gradle.kts:22]: Looks good. Looks like that's why the checks are failing. Maybe you should go ahead and publish so the tests in the PR pass?

File: readme.md (hunk 2 above)
Comment (ds58) [reply readme.md:140]: When running on a native arm platform you don't need to use the cross compilation toolchain

File: build.gradle.kts (hunk 1 above)
Comment (ds58) [reply build.gradle.kts:22]: Sure, note that there is 1 failing test on develop already

**Review by reesep5 — APPROVED** [review]

**Review by calvertdw — APPROVED** [review]

**Pull request merged**